spec/API_specification/signatures/constants.py

```diff
@@ -0,0 +1,25 @@
+e = None
```

Comment:
```suggestion
e: float = NotImplemented
```
Might i suggest `NotImplemented` here as sentinel value over `None`, as the former is (as far as type checkers are concerned) is equivalent to `typing.Any`. 

Or maybe just drop the assignment in its entirety? Assuming the docstring can still be parsed in such situation.
```suggestion
e: float
```

Reply:
Thanks for the feedback! I'm also not a fan of `None`, the issue is that we need to assign it for sphinx to render it as we want. I think `NotImplemented` is a good option